.github/workflows/plgd-device-test-with-cfg.yml

```diff
@@ -32,7 +32,7 @@ on:
         required: false
         default: false
 env:
-  TEST_CLOUD_SERVER_IMAGE: ghcr.io/plgd-dev/device/test-cloud-server:latest
+  TEST_CLOUD_SERVER_IMAGE: ghcr.io/plgd-dev/device/test-cloud-server:system-clock
```

Comment:
Hmm, when I switch back to latest, I'll have to disable the new tests I've added because they only work with plgd-time enabled. I guess we can reenable them once this feature is approved and merged.